pkg/collector/corechecks/oracle/tablespaces_test.go

```diff
@@ -37,4 +37,9 @@ func TestTablespaces(t *testing.T) {
 	tags := []string{fmt.Sprintf("pdb:%s", expectedPdb), "tablespace:TBS_TEST"}
 	s.AssertMetricOnce(t, "Gauge", "oracle.tablespace.size", 104857600, c.dbHostname, tags)
 	s.AssertMetricOnce(t, "Gauge", "oracle.tablespace.offline", 0, c.dbHostname, tags)
+	_, err := c.db.Exec("ALTER TABLESPACE TBS_TEST OFFLINE")
+	err := c.Run()
+	require.NoError(t, err)
+	s.On("Gauge", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return()
+	s.AssertMetricOnce(t, "Gauge", "oracle.tablespace.offline", 1, c.dbHostname, tags)
```

Comment:
Can we put the tablespace `online` again to have a clean baseline for the new test? Alternatively, we can do that prior to testing if the tablespace is `offline`